<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="XE-T-NITINGO" name="Test" tests="6" failures="1" timestamp="29 Aug 2016 12:02:03 GMT" time="0.197" errors="0">
  <testcase name="testcase1" time="0.048" classname="Xebia.SampleMavenProject.TestNGTestClass1"/>
  <testcase name="testcase2" time="0.008" classname="Xebia.SampleMavenProject.TestNGTestClass1">
    <skipped/>
  </testcase> <!-- testcase2 -->
  <testcase name="testcase2" time="0.008" classname="Xebia.SampleMavenProject.TestNGTestClass1">
    <skipped/>
  </testcase> <!-- testcase2 -->
  <testcase name="testcase2" time="0.008" classname="Xebia.SampleMavenProject.TestNGTestClass1">
    <skipped/>
  </testcase> <!-- testcase2 -->
  <testcase name="testcase2" time="0.008" classname="Xebia.SampleMavenProject.TestNGTestClass1">
    <skipped/>
  </testcase> <!-- testcase2 -->
  <testcase name="testcase2" time="0.009" classname="Xebia.SampleMavenProject.TestNGTestClass1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at Xebia.SampleMavenProject.TestNGTestClass1.testcase2(TestNGTestClass1.java:21)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testcase2 -->
</testsuite> <!-- Test -->
